package.json

```diff
@@ -25,16 +25,30 @@
     "@popperjs/core": "^2.11.6",
     "@tanstack/react-query": "^5.14.1",
     "@tanstack/react-table": "^8.7.9",
-    "axios": "^1.3.2",
+    "axios": "^1.8.3",
```

Comment:
This seems like a lot of additional dependencies. We be good to double check that all of these are really needed